### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by zewdu444**
## Bookstore: styling

- [x] Followed ([Zeplin](https://zeplin.io/)) guide line 
- [x] Styling UI using Prime React 
- [x] Modify readme file
- [x] Deploy to render https://bookstore-beoz.onrender.com/

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,27 +1,4 @@
...
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    "Courier New",
-    monospace;
+  background-color: var(--surface-100);
```
Comment (tufoinnkuo10) [inline src/index.css:3]: - [ ] please kindly ensure that the styling is almost exact like the Zeplin version. nice job. please the add book button should be aligned with the page like in the original version.
yours
Screenshot (848)_LI
Zeplin
Screenshot (847)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @zewdu444 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Gitflow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ style well followed

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/index.css (hunk 1 above)
Comment (zewdu444) [reply src/index.css:3]: thanks  updated based on the feedback

**Review by prolajumokeoni — APPROVED** [review]
Hi @zewdu444 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
  - Your design matches Zephlin's design ✅ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**